app/syncables_test.go

```diff
@@ -297,14 +296,15 @@ func TestCreateDefaultMemberships(t *testing.T) {
 		t.Errorf("unable to add user to channel: %s", err.Error())
 	}
 
-	timeAfterLeaving := model.GetMillis()
+	timeAfterLeaving := model.GetMillis() + 1
```

Comment:
Would it make sense to do `timeAfterLeaving := timeBeforeLeaving + 1`? Or some such. Just to make the dependency clear.

Reply:
I don't think that would fix the issue though cause we need to wait for `LeaveChannel` to have executed.

Reply:
Hmm .. but isn't it sufficient to have `timeAfterLeaving` greater than `timeBeforeLeaving`? Anyways, the current way is also fine.

Reply:
I don't think so because what we end up checking on is the `ChannelMemberHistory.LeaveTime` field is set in `ChannelMemberHistoryStore.LogLeaveEvent()` which is eventually called by `App.LeaveChannel()` so we can't really control what that value is going to be.

Reply:
Got it. Thanks for clearing that up.